mailphax.rb

```diff
@@ -2,62 +2,170 @@
 require 'phaxio'
 require 'mail'
 require 'pony'
+require 'tempfile'
+require 'openssl'
+require 'to_regexp'
 
-if not ENV['PHAXIO_KEY'] or not ENV['PHAXIO_SECRET']
-  raise "You must specify your phaxio API keys in PHAXIO_KEY and PHAXIO_SECRET"
+if not ENV['PHAXIO_KEY'] or not ENV['PHAXIO_SECRET'] or not ENV['MAILGUN_KEY']
```

Comment:
I see now that you addressed this in your comments and removed the other mail providers.  you're probably right.  we don't have the man power right now to implement the other providers unless they're requested.